dallinger/experiment.py

```diff
@@ -1429,3 +1429,38 @@ def load():
     except ImportError:
         logger.error("Could not import experiment.")
         raise
+
+
+EXPERIMENT_TASK_REGISTRATIONS = {}
+
+
+def scheduled_task(trigger, **kwargs):
+    """Creates a decorator to register experiment functions or classmethods as
+    tasks for the clock process. Accepts all
+    arguments for `apscheduler.schedulers.base.BaseSchedule.scheduled_job`
+    The task registration is deferred until clock server setup to allow tasks to be
+    overridden by subclasses.
+
+    :param trigger: an ``appscheduler`` trigger type. One of "interval", "cron",
+                    or "date"
+    :param \**kwargs: other arguments for `apscheduler.schedulers.base.BaseSchedule.scheduled_job`
+                      generally used for trigger arguments to determine
+                      the run interval.
+
+    :returns: A decorator to register methods from a class as scheduled tasks.
+    """  # noqa
```

Comment:
What was the `# noqa` for?

Reply:
Flake 8 doesn't like the `\**kwargs` (the backslash freaks it out, but it's what sphinx wants).